scripts/nice2/helpers.sh

```diff
@@ -103,9 +103,20 @@ function waitForWidgetServer() {
   echo "widget-server is ready"
 }
 
-function emptyDB() {
+function setupDB() {
```

Comment:
question: What is the difference between `setupDB` and `restoreDB`? 

Couldn't we just use one of those?

Reply:
nothing anymore :)